sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -101,4 +126,46 @@ export class AssessmentAESTController extends BaseController {
       true,
     );
   }
+
+  /**
+   * Triggers manual reassessment for an application.
+   * Application cannot be archived or in any of the statuses 'Cancelled', 'Overwritten' or 'Draft' and original assessment must be in completed status.
+   * @param payload request payload.
+   * @param applicationId application id.
+   * @returns id of the assessment created.
+   */
+  @Roles(Role.AESTManualTriggerReassessment)
+  @Post(":applicationId/manual-reassessment")
```

Comment:
Since it is not in this controller please follow the same from other ednpoints, it should be
```ts
@Post("application/:applicationId/manual-reassessment")
```